assets/js/ga-integration.js

```diff
@@ -0,0 +1,5 @@
+jQuery(document).ready( function($) {
+    $(document).on( "found_variation", "form.cart", function( e, variation ) {
+        google_analytics_integration = variation.google_analytics_integration;
```

Comment:
It's a good practice to avoid polluting the global scope. Such a variable could me accidentally, or maliciously changed by some other entity.
I'd rather limit its availability, either by attaching it to some element, object, or sharing a JS scope with

Reply:
As we discussed, it's now storing the [data in an array](https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/230/commits/e424a53bcf7e4dc60091db7ef0b46777c4fbb49f) and then retrieves the correct data based on the currently selected variation when a product is added to a cart. It's not the most elegant solution but as @mikkamp said, when the extension moves toward a generic JS function for event tracking we'll be able to avoid globals.